dispatchers/dispatchers.go

```diff
@@ -227,6 +230,9 @@ func (dS *DispatcherService) V1GetProfilesForEvent(ev *utils.CGREvent,
 	retDPfl, errDpfl := dS.dispatcherProfilesForEvent(tnt, ev, utils.MapStorage{
 		utils.MetaReq:  ev.Event,
 		utils.MetaOpts: ev.APIOpts,
+		utils.MetaVars: utils.MapStorage{
+			utils.MetaMethod: ev.APIOpts[utils.MetaMethod],
```

Comment:
This exposes some insecurity, please take it out. We should only hardcode it for now.